src/login/LoginView.js

```diff
@@ -41,7 +42,7 @@ export class LoginView {
 	invalidCredentials: boolean;
 	accessExpired: boolean;
 	savePassword: Checkbox;
-	appButtons: Button[];
+	appButtons: ButtonAttrs[];
```

Comment:
We don't need to save them into array, just render them inline, maybe in a separate method